src/bp/core/services/middleware/event-collector.ts

```diff
@@ -12,6 +12,57 @@ import ms from 'ms'
 import { SessionIdFactory } from '../dialog/session/id-factory'
 
 type BatchEvent = sdk.IO.StoredEvent & { retry?: number }
+export const LAST_EVENT_STEP = 'completed'
```

Comment:
De we have possible step status type ?

Reply:
The first is `received` and the last is `completed`, maybe it can be changed for `finished` ?

Reply:
I know but do we have a type declaration for this somewhere ?

Reply:
Changed it to a more usable object